source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -1,23 +1,22 @@
 #include "CGLTFMeshFileLoader.h"
-#include "CMeshBuffer.h"
+#include "CSkinnedMesh.h"
 #include "coreutil.h"

```

Comment:
These should be switched.

Reply:
Why? If `CSkinnedMesh.h` forgot to include `coreutil.h`, wouldn't that hide the issue (assuming we were the only consumer)?

Reply:
That is a good observation, but keeping track of which headers might need to include which other headers at that level of exactness is probably not worth it. Within a group of headers, I usually order them alphabetically.

If we want to do it based on the dependency tree, then we should change it for all the includes.

Reply:
That is a good observation, but within a group of headers I usually order them alphabetically, because that level of exactness seems like it wouldn't be worth the effort. It's hard to know for example which order the standard library includes depend on each other.